app/Repositories/SignupRepository.php

```diff
@@ -15,13 +15,15 @@ class SignupRepository
      */
     public function create($data)
     {
+        // Create the post_signup event
         $event = Event::create([
             'northstar_id' => $data['northstar_id'],
-            'event_type' => 'signup',
+            'event_type' => 'post_signup',
```

Comment:
What is `post_signup` supposed to signify? If this is just for creating a signup I think we should not use the "post" terminology cause it seems confusing. A post is different than a signup